lib/sensors/i2c_sensors.hpp

```diff
@@ -6,17 +6,21 @@
 
 namespace hyped::sensors {
 
+enum class I2cConfigureResult { kFailure = 0, kSuccess };
+
```

Comment:
Idk if this is a standardized thing I'm missing here but what is `T`. Like what would it typically represent if you made a class inheriting from this one?

Reply:
Template - so it can be any type. The whole point of this was to be able to return different types when you call `readAllChannels` in mux depending on the sensor.